src/Model/LanguageFeatures.re

```diff
@@ -134,10 +134,9 @@ let requestDocumentHighlights =
   |> DocumentHighlightProvider.request((buffer, location));
 };
 
-let requestCompletions =
-    (~buffer: Buffer.t, ~meet: CompletionMeet.t, ~location: Location.t, lf: t) => {
+let requestCompletions = (~buffer: Buffer.t, ~meet: CompletionMeet.t, lf: t) => {
   lf.completionProviders
-  |> CompletionProvider.request((buffer, meet, location));
+  |> CompletionProvider.request((buffer, meet, meet.location));
```

Comment:
The `location` argument seems like unnecessary duplication, since it's contained in `meet` and I can't see any good reason to request completions at a location other than the meet. `meet` and `location` a re carried together through a whole bunch of boundaries, and is unnecessary the entire way, but this was the simplest change from a user POV.